dao/src/main/java/cz/idomatojde/dao/UserDao.java

```diff
@@ -1,9 +1,33 @@
 package cz.idomatojde.dao;
 
 import cz.idomatojde.dao.common.BaseDAO;
+import cz.idomatojde.entity.TimetableChatMessage;
```

Comment:
Same unused import? Have not pulled it and checked in IDEA but seems like it is not used.